ant/pom.xml

```diff
@@ -17,8 +17,8 @@
 			<scope>provided</scope>
 		</dependency>
 		<dependency>
-			<groupId>com.sun.activation</groupId>
-			<artifactId>javax.activation</artifactId>
+			<groupId>jakarta.activation</groupId>
+			<artifactId>jakarta.activation-api</artifactId>
```

Comment:
for activation v 2.0.x, I'd avoid changing this as it can do more harm than good - `com.sun.activation` may conflict with `jakarta.activation` on JPMS. This change is fine with activation-api 2.1.0 (currently RC1); an implementation is ie [angus-activation](https://github.com/eclipse-ee4j/angus-activation), currently at 1.0.0-M1

Reply:
It worked with your suggestion.  But at least for my build, it worked with what I have also.